static/roomday.js

```diff
@@ -164,12 +166,18 @@ function sendTalkState(talk) {
   const start = parseTimeInput(talk.querySelector(".talk-time-start .talk-time-input").value);
   const end = parseTimeInput(talk.querySelector(".talk-time-end .talk-time-input").value);
   const editStatus = talk.querySelector("input[name='edit-status-" + talkId + "']:checked").value;
+  const thumbnail = parseTimeInput(talk.querySelector(".talk-time-thumbnail .talk-time-input").value);
+
+  if (thumbnail > end || thumbnail < start) {
+    alert("Invalid thumbnail selection. Make sure the thumbnail time occurs during the talk.");
```

Comment:
Try this out on a fresh talk - since thumbnail starts at 0, this pop-up will occur every time start/end is adjusted, until the editor puts in a time for it.

One alternative UI may be to just go ahead and clamp the value between start and end, if start <= end. The editor will probably figure out how it works soon enough. Or make the box red to indicate it's currently an invalid value.

Reply:
I changed it so if the thumbnail time is outside of the talk time, it automatically sets the thumbnail time to the start time.

Reply:
It's a little bit surprising when it gets clipped by a changed end time but jumps to the start, but not a blocker for this PR.